antenna-sw360-adapter/src/main/java/org/eclipse/sw360/antenna/sw360/adapter/SW360UserClientAdapter.java

```diff
@@ -18,13 +18,19 @@
 
 public class SW360UserClientAdapter {
     private String restUrl;
+    private boolean proxyEnable;
+    private String proxyHost;
+    private int proxyPort;
 
-    public SW360UserClientAdapter(String backendURL) {
+    public SW360UserClientAdapter(String backendURL, boolean proxyEnable, String proxyHost, int proxyPort) {
         restUrl = backendURL;
+        this.proxyEnable = proxyEnable;
```

Comment:
This adapter should also directly create the client in the constructor (similar to the other adapters) and not save the configuration in fields.